src/Symfony/Bundle/FrameworkBundle/.gitignore

```diff
@@ -1,3 +1,4 @@
 vendor/
 composer.lock
 phpunit.xml
+Tests/Fixtures/micro-app/var/
```

Comment:
Instead, can't we remove the var folder after the test cases? I think we already do so for other similar tests.